gordon-plugin/src/main/kotlin/com/banno/gordon/TestDistributor.kt

```diff
@@ -132,13 +133,16 @@ private fun CoroutineScope.runTestsInPool(
     instrumentationRunnerOptions: InstrumentationRunnerOptions,
     testTimeoutMillis: Long,
     devices: List<JadbDevice>,
-    testDistributor: TestDistributor
+    testDistributor: TestDistributor,
+    totalTests: Int? = null
 ): List<Deferred<Map<TestCase, TestResult>>> {
+    var index = 0f
```

Comment:
I don't see any reason for this to be a float

```suggestion
    var index = 0
```